### PR TITLE
Added Swift heatmap demo.

### PR DESCRIPTION
**Pull request by willum070**

### CODE REVIEW

File: samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        }
+      }
+    } catch {
+      print(error.localizedDescription)
+    }
+    // Add the latlngs to the heatmap layer.
+    heatmapLayer.weightedData = list
+  }
+
+  func randomScale() -> Double {
```
Comment (mountainvat) [inline samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift]: Please remove this function

File: samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import Foundation
+import GoogleMaps
+import UIKit
+
+class HeatmapViewController: UIViewController, GMSMapViewDelegate {
+  private var mapView: GMSMapView!
+  private var heatmapLayer: GMUHeatmapTileLayer!
+  private var button: UIButton!
+
+  private var heatmapItemCount = 10000
```
Comment (mountainvat) [inline samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift]: Please remove the 3 variables. They seem unused.

File: samples/SwiftDemoApp/SwiftDemoApp/AppDelegate.swift [hunk 3]
```diff
@@ -17,7 +17,7 @@ import GoogleMaps
 import UIKit
 
 // Change this key to a valid key registered with the demo app bundle id.
-let kMapsAPIKey = ""
+let kMapsAPIKey = "YOUR-API-KEY"
```
Comment (mountainvat) [inline samples/SwiftDemoApp/SwiftDemoApp/AppDelegate.swift]: We have a check down below for an empty key and print out an error message to the user. So please either revert this change or update the check.

File: samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift (hunk 2 above)
Comment (willum070) [reply samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift]: heatmapItemCount does not belong, the other variables are actually used.

File: samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+import Foundation
+import GoogleMaps
+import UIKit
+
+class HeatmapViewController: UIViewController, GMSMapViewDelegate {
+  private var mapView: GMSMapView!
+  private var heatmapLayer: GMUHeatmapTileLayer!
+  private var button: UIButton!
+
+  private var cameraLatitude = -37.848
```
Comment (mountainvat) [inline samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift]: I still cannot see where these variables are used. The camera variable in loadView() uses the constant lat, lng in its constructor.
